src/SUMMARY.md

```diff
@@ -5,10 +5,16 @@
 - [Testing Policies](./testing-policies.md)
 - [Writing Policies](./writing-policies/index.md)
   - [Rust](./writing-policies/rust.md)
+  - [Go](./writing-policies/go/01-intro.md)
+    - [Create a new policy](./writing-policies/go/02-scaffold.md)
+    - [Define policy settings](./writing-policies/go/03-policy-settings.md)
+    - [Write the validation logic](./writing-policies/go/04-valitation.md)
```

Comment:
```suggestion
    - [Write the validation logic](./writing-policies/go/04-validation.md)
```

Reply:
Note well: renaming will create a new file and leave the old one around.

Reply:
I fixed that manually. Thanks